engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/new/new-repository.controller.js

```diff
@@ -45,6 +45,14 @@ angular.module('Bastion.repositories').controller('NewRepositoryController',
             }
         }
 
+        Setting.get({"search": "name = enable_deferred_download_policies"},
+        function (data) {
+            data = angular.fromJson(data);
+            $scope.enableDownloadPolicies = data.results[0].value;
+        }, function (data) {
+            ApiErrorHandler.handleGETRequestErrors(data, $scope);
```

Comment:
@ehelms I wasn't sure if this is the correct way to handle an error from the API?